stripe/src/main/java/com/stripe/android/net/SourceRetriever.java

```diff
@@ -0,0 +1,17 @@
+package com.stripe.android.net;
+
+import android.support.annotation.NonNull;
+
+import com.stripe.android.exception.StripeException;
+import com.stripe.android.model.Source;
+
+/**
+ * Represents something that can retrieve a source.
+ */
+interface SourceRetriever {
```

Comment:
Used SourceRetriever in both the sync and async polling versions, so I pulled it out into a class